openmetadata-ui/src/main/resources/ui/src/styles/app.less

```diff
@@ -769,3 +769,8 @@ a[href].link-text-grey,
   width: 20px;
   height: 20px;
 }
+
+.ant-input,
+.ant-input-password-icon.anticon {
+  color: #757575;
+}
```

Comment:
i don't think we should be overriding the entire ant-input.
Can you check the documentation of Antd Input. There should be a variable based on which color is passed to the input. We can override that variable in variables.less file.
Also directly providing color code 757575 is not a good idea, we should be using variables.